### ISSUE TITLE
dist cleanup

### ISSUE THREAD

**Issue by rkitover** [avar/dist-zilla-pluginbundle-avar#2]
Hey @avar,

so I needed to upload a dist today after I dunno how many years.

Anyway, we need to fix a few minor issues with this particular dist, and I'm happy to take care of that, just wanted to discuss with you first.

@haarg says 0.32 is the last version indexed on CPAN, while you uploaded 0.33 and 0.34:

https://rt.cpan.org/Ticket/Display.html?id=149212

this is indeed what metacpan says. I have no idea how this happened right now, hopefully uploading a new dist will index it properly.

Also, smartmatch and given/when are apparently deprecated now:

```console
v GENTOO ~/source/repos/dist-zilla-pluginbundle-avar [master|v]
rkitover@epyc > perl -MDist::Zilla::PluginBundle::AVAR -E 'say $Dist::Zilla::PluginBundle::AVAR::VERSION'
Smartmatch is deprecated at (eval 175) line 8.
Smartmatch is deprecated at (eval 175) line 15.
Smartmatch is deprecated at (eval 176) line 8.
Smartmatch is deprecated at (eval 176) line 15.
Smartmatch is deprecated at (eval 177) line 8.
Smartmatch is deprecated at (eval 177) line 15.
Smartmatch is deprecated at (eval 178) line 11.
Smartmatch is deprecated at (eval 178) line 21.
given is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 62.
when is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 63.
when is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 64.
given is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 71.
when is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 72.
given is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 82.
when is deprecated at /home/rkitover/perl5/lib/perl5/Dist/Zilla/PluginBundle/AVAR.pm line 83.
0.32
```

. Let me know what you think. Hope you're doing well.

**Comment by haarg** [comment]
smartmatch and given/when were already fixed in this repo for the 0.33 release.

**Comment by rkitover** [comment]
smartmatch is still apparently broken:

```console
v GENTOO ~/source/repos/dist-zilla-pluginbundle-avar/Dist-Zilla-PluginBundle-AVAR-0.35 [master|v]
rkitover@epyc > perl -MDist::Zilla::PluginBundle::AVAR -E 'say $Dist::Zilla::PluginBundle::AVAR::VERSION'
Smartmatch is deprecated at (eval 175) line 8.
Smartmatch is deprecated at (eval 175) line 15.
Smartmatch is deprecated at (eval 176) line 8.
Smartmatch is deprecated at (eval 176) line 15.
Smartmatch is deprecated at (eval 177) line 8.
Smartmatch is deprecated at (eval 177) line 15.
Smartmatch is deprecated at (eval 178) line 11.
Smartmatch is deprecated at (eval 178) line 21.
0.35
```

**Comment by haarg** [comment]
Those are coming from Syntax::Keyword::Junction via Moose::Autobox. This module is loading Moose::Autobox, but doesn't use anything from it. Fixed in #3

**Comment by rkitover** [comment]
Ok, 0.35 has been indexed. Thanks @haarg.
